doc/source/examples/pdp_regression_bike.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>they returned -&gt; they return</p><p> </p><p>Maybe worth clarifying what "different probability distributions" means and how it effects the PD values? Perhaps there is a link to a <code>sklearn</code> doc page? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>